<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Z-Server</string>
  </property>
  <widget class="QWidget" name="Centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="Main_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QGroupBox" name="Clients_groupBox">
         <property name="title">
          <string>Клиенты</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QListWidget" name="Clients_listWidget">
            <property name="autoScroll">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::IgnoreAction</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Chat_groupBox">
      <property name="title">
       <string> Входной чат</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="Chat_textBrowser">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Chat_lineEdit">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="File_menu">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="Help_menu">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="About_action"/>
   </widget>
   <widget class="QMenu" name="Net_menu">
    <property name="title">
     <string>Сеть</string>
    </property>
    <addaction name="StartStop_action"/>
   </widget>
   <addaction name="File_menu"/>
   <addaction name="Net_menu"/>
   <addaction name="Help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="About_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="StartStop_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пуск/Стоп</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
